src/commands/db/create.ts

```diff
@@ -0,0 +1,259 @@
+import ora from 'ora';
+import inquirer from 'inquirer';
+import Command from '../../base.js';
+import { Flags } from '@oclif/core';
+import { createDebugLogger } from '../../utils/output.js';
+import spacing from '../../utils/spacing.js';
+import { ux } from '@oclif/core';
+import { string } from '@oclif/core/lib/flags.js';
+import { relativeTimeThreshold } from 'moment';
+
+export default class Create extends Command {
+  static description = 'create a new database';
+
+  static PATH = 'v1/databases';
+
+  static flags = {
+    ...Command.flags,
+    name: Flags.string({
+      char: 'n',
+      description: 'name of your database',
+    }),
+    plan: Flags.string({
+      description: 'plan',
+    }),
+    network: Flags.string({
+      description: 'use public network or not',
+    }),
+    type: Flags.string({
+      char: 't',
+      description: 'choose which database to use',
+    }),
+    version: Flags.string({
+      char: 'v',
+      description: 'version of the database',
+    }),
+    yes: Flags.boolean({
+      char: 'y',
+      description: 'say yes to continue prompt',
+    }),
+  };
+
+  async run(): Promise<void> {
+    this.spinner = ora();
+
+    const { flags } = await this.parse(Create);
+    const debug = createDebugLogger(flags.debug);
+
+    await this.setGotConfig(flags);
+    const account = await this.getCurrentAccount();
+
+    ((account && account.region === 'germany') || flags.region === 'germany') &&
+      this.error('We do not support germany any more.');
+
+    const hostname = flags.name || (await this.promptHostname());
+    const type = flags.type || (await this.promptType());
+    const version = flags.version || (await this.promptVersion(type));
+    const publicNetwork =
+      flags.network === 'public'
+        ? true
+        : flags.network === 'private'
+        ? false
```

Comment:
You can just do this:

```suggestion
    const publicNetwork = flags.network === 'public';
```

Reply:
In that way, anything other than 'public' is considered as private. Is it OK?

Reply:
oclif flags can have a limited allowed list of values. checkout the `builder-location` flag in the `deploy` command. it only accepts `iran | germany` as values.